server/chat.ts

```diff
@@ -152,14 +152,13 @@ class PatternTester {
 
 // These classes need to be declared here because they aren't hoisted
 class MessageContext {
+	readonly user: User;
+	readonly language: string | null;
```

Comment:
idk it makes sense that a command would want to modify this.

Reply:
Like if someone does `/setlanguage English` the response should probably be in English.